test/integrations/component.test.ts

```diff
@@ -34,6 +38,68 @@ afterAll(async () => {
   await createHttpTerminator({ server }).terminate();
 });
 
+// unmock already existing axios-mocking
+jest.unmock('axios')
+
+jest.mock('axios')
```

Comment:
https://www.npmjs.com/package/axios-mock-adapter this looks interesting, should we try this?

Reply:
Used this and implemented the network requests mocking

Reply:
Hence resolving this comment